Allow extensions to / modifications of wxUniversal themes
by exporting the wxDelegateTheme class

Signed-off-by: Anders Larsen <al@alarsen.net>
---

 include/wx/univ/theme.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)


diff --git a/include/wx/univ/theme.h b/include/wx/univ/theme.h
index 95544b4..4694f37 100644
--- a/include/wx/univ/theme.h
+++ b/include/wx/univ/theme.h
@@ -83,7 +83,7 @@ private:
 // will be left to the original theme
 // ----------------------------------------------------------------------------
 
-class wxDelegateTheme : public wxTheme
+class WXDLLIMPEXP_CORE wxDelegateTheme : public wxTheme
 {
 public:
     wxDelegateTheme(const wxString& theme);
